### PR TITLE
Release v0.2.8

### PR DESCRIPTION
**Pull request by josephlr**
Also updates the changelog

We should probably release this before the major changes in #291

Signed-off-by: Joe Richey <joerichey@google.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.8] - 2022-10-20
+### Added
+
+- [Web Crypto API] can now be used on Node.js if avalible [#284]
```
Comment (newpavlov) [inline CHANGELOG.md:10]: The new lines after subsection are redundant and not consistent with entries for other versions.

**Review by newpavlov — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (briansmith) [reply CHANGELOG.md:10]: This, and the other documentation, leave it ambiguous about which API will be used when both are detected. The big table in lib.rs's module doc is actively misleading, if I am understanding correctly that the webcrypto API is preferred if detected.

Comment (josephlr) [reply CHANGELOG.md:10]: That's fair. I've fixed the spacing and spelling issues. I've also moved the WebCrypto stuff to the `Changed` section and actually elaborated on what we changed.

Let me know if it looks good to y'all.

I've also opened #295 to update the table in the Rust Docs.

**Review by maccesch — APPROVED** [review]

**Comment by josephlr** [comment]
Merging this, now that #295 is merged.

**Comment by josephlr** [comment]
Tag: https://github.com/rust-random/getrandom/releases/tag/v0.2.8
Published: https://crates.io/crates/getrandom/0.2.8

**Pull request merged**